quickstart/kubernetes/index.md

```diff
@@ -5,79 +5,152 @@ redirect_from: /reference/kubernetes.html
-
-> **Note:** The Pulumi provider currently supports Kubernetes 1.5+.  If you have a specific version
-> requirement and are unsure of whether we currently support it, or are certain we don't and need us
-> to begin doing so, please contact us.
-
-## Setup
-
-By default, Pulumi uses the same configuration as `kubectl`, so if you're already connected to a cluster, Pulumi will
-"just work." For more details, including this initial setup, please see the [Kubernetes setup page](./setup.html).
-
-## Tutorials
-
-The easiest way to start with Kubernetes is to follow one of the tutorials:
-
-* Getting your feet wet [deploying a container to Kubernetes and exposing it to the Internet with a
-    `Service`](./tutorial-exposed-deployment.html). This example shows off several unique features of
-    Pulumi, including diffing and incremental errors when deployment fails.
-* ["Adopting" a Helm Chart](./tutorial-wordpress-chart.html). This example additionally demonstrates
-   the rich status updates Pulumi offers, allowing you to reliably and reproducibly deploy code,
-   even if it isn't yours.
-* [The canonical Kubernetes Guestbook example](./tutorial-guestbook.html). You can see the original
-   example [here](https://github.com/pulumi/examples/tree/master/kubernetes-ts-guestbook) for
-   comparison.
-* [Triggering a new deployment when changing the data in a
-   `ConfigMap`](./tutorial-configmap-rollout.html).
-   This example shows how Pulumi will safely allow you to update containers to use new configuration
-   data in a way that is natural to the user.
-* [Staged rollout gated on Prometheus health checks](./tutorial-p8s-rollout.html). This example
-  shows how we can roll out to a small canary ring, check Prometheus metrics, and then roll out to a
-  larger prod ring only when that succeeds.
-* [Run a Stateless App Deployment](./tutorial-stateless-app.html): Deploy a simple stateless Nginx web server backed
-    by a Docker container
+[Kubernetes][k8s] is an open source platform for running containerized applications on a cluster of
+machines. Applications are managed through a RESTful API that exposes mechanisms to deploy, scale,
+and introspect on resources in the cluster.
+
+Pulumi is an infrastructure-as-code tool that exposes a Kubernetes SDK that allows users to write
+Kubernetes applications in the language of their choice, such as JavaScript and Python. It is
+designed to fit in anywhere you already use Kubernetes:
+
+-   **API-compatible with Kubernetes** -- users do **not** have to learn a new API to write
+    Kubernetes applications with Pulumi. (See the [Kubernetes][api-reference] and
+    [Pulumi](/reference/pkg/nodejs/@pulumi/kubernetes/index.html) API documentation for more
+    details.)
+-   **Compatible with Kubernetes YAML and Helm Charts.** We will see examples of this in the
+    subsequent sections.
+-   **Drop-in replacement for `kubectl`.** Anywhere you have a [kubeconfig file][kubeconfig], you
```

Comment:
I'm not sure this is true ... You still need to use `kubectl` for many things ...